driver/pace/driver/diagnostics.py

```diff
@@ -177,7 +177,7 @@ def _compute_column_integral(
     if len(q_in.shape) < 3:
         assert ShapeError(f"{name} does not have vertical levels.")
     column_integral = pace.util.Quantity(
-        sum(
+        q_in.np.sum(
             q_in.data[:, :, k] * delp.data[:, :, k]
             for k in range(q_in.metadata.extent[2])
         ),
```

Comment:
I think it will only be faster if you do something like
```suggestion
        q_in.np.sum(
            q_in.data[:, :, q_in.origin[2]:q_in.origin[2] +q_in.extent[2]] * delp.data[:, :, q_in.origin[2]:q_in.origin[2] +q_in.extent[2]]
        ),
```
You should consider putting the weird slice thing in a `k_slice = slice(q_in.origin[2], q_in.origin[2] + q_in.extent[2]` object and adding a `if q_in.dims[2] != Z_DIM: raise NotImplementedError("this function assumes the z-dimension is the third dimension")`